examples/jsm/webxr/XRHandModelFactory.d.ts

```diff
@@ -0,0 +1,28 @@
+import {
+	Group,
+	Object3D,
+} from '../../../src/Three';
+
+import { GLTFLoader } from '../loaders/GLTFLoader';
+
+export class XRHandModel extends Object3D {
+
+	constructor();
+
+	motionController: any;
+
+}
+
+export class XRControllerModelFactory {
```

Comment:
This looks like a copy/paste error. Shouldn't you define here `XRHandModelFactory`?

Reply:
Yes, I'll get on it. Modified the controller types to fit.